File: csharp/Types/Verification.cs
using Newtonsoft.Json;

namespace Authdog.Types
{
    /// <summary>
    /// Email verification status
    /// </summary>
    public class Verification
    {
        [JsonProperty("id")]
        public string Id { get; set; } = string.Empty;

        [JsonProperty("email")]
        public string Email { get; set; } = string.Empty;

        [JsonProperty("verified")]
        public bool Verified { get; set; }

        [JsonProperty("createdAt")]
        public string CreatedAt { get; set; } = string.Empty;

        [JsonProperty("updatedAt")]
        public string UpdatedAt { get; set; } = string.Empty;
    }
}

File: planned/php/src/Types/Verification.php
<?php

namespace Authdog\Types;

/**
 * Email verification status
 */
class Verification
{
    public string $id;
    public string $email;
    public bool $verified;
    public string $createdAt;
    public string $updatedAt;

    public function __construct(array $data)
    {
        $this->id = $data['id'] ?? '';
        $this->email = $data['email'] ?? '';
        $this->verified = $data['verified'] ?? false;
        $this->createdAt = $data['createdAt'] ?? '';
        $this->updatedAt = $data['updatedAt'] ?? '';
    }
}
